src/Board.tsx

```diff
@@ -1,34 +1,21 @@
-import * as React from 'react';
+import React, { FunctionComponent } from 'react';
 import { Square } from './Square';
 
 type Props = {
   squares: string[];
+  clickHandler: Function;
 };
 
-export class Board extends React.Component<Props> {
-  public constructor(props: Props) {
-    super(props);
-  }
+export const Board: FunctionComponent<Props> = (props) => {
```

Comment:
I'm not a big fan of spec'ing `FunctionComponent` explicitly (props: Props) feels cleaner